### ISSUE TITLE
Transparent Background Option

### ISSUE THREAD

**Issue by Jesuite** [ChurchApps/FreeShow#777]
**Describe the feature**
Would it be possible to have transparent as a background option for templates with NDI, just a dedicated option like the rest of the background color or is there already a way to go this without NDI?

**Additional context**

**Comment by Bagel94** [comment]
Yes, you can already do this. In settings, go to outputs. Then on the output you want NDI, there is a transparent setting once NDI has been enabled?
![image](https://github.com/user-attachments/assets/bdc29e51-7212-40d9-9712-1b8d57c36bba)

**Comment by Jesuite** [comment]
Yeah I'm aware of that option, I'm talking about an option without NDI

**Comment by vassbo** [comment]
Please explain the use case for this.

**Comment by Jesuite** [comment]
In my use case it would probably be if I want my output screen to be a layer Input in my streaming software like for layering lower thirds lyrics over a camera view. I tried using NDI but it splits the bandwidth and it causes a lot of lagging for both the camera and 
overlay text from FreeShow. So, what I resorted to was using a green screen to get transparent background but I would be a lot easier if there was a transparent background option instead of using a the whole green screen process.
This is my use case I'm not sure if anyone uses FreeShow this way.
